packages/salesforcedx-vscode-core/test/vscode-integration/util/authInfo.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+import { nls } from '../../../src/messages';
+import { ConfigUtil, OrgAuthInfo } from '../../../src/util';
+
+// tslint:disable:no-unused-expression
+describe('getDefaultDevHubUsernameOrAlias', () => {
+  it('should return notification if there is no dev hub set', async () => {
+    const configUtilStub = sinon.stub(ConfigUtil, 'getConfigValue');
+    configUtilStub.returns(undefined);
+    const infoMessageStub = sinon.stub(vscode.window, 'showInformationMessage');
+
+    await OrgAuthInfo.getDefaultDevHubUsernameOrAlias(true);
+
+    expect(infoMessageStub.calledOnce).to.be.true;
+    configUtilStub.restore();
+    infoMessageStub.restore();
+  });
+  it('should run authorize a dev hub command if button clicked', async () => {
+    const configUtilStub = sinon.stub(ConfigUtil, 'getConfigValue');
+    configUtilStub.returns(undefined);
+    const authDebvHubStub = sinon.stub(vscode.window, 'showInformationMessage');
```

Comment:
I'd change the name of this to reflect the functionality more. Something like `showMessageStub`